src/cargo/core/compiler/job_queue/mod.rs

```diff
@@ -952,11 +952,9 @@ impl<'cfg> DrainState<'cfg> {
                 }
 
                 for warning in output.warnings.iter() {
-                    let warning_with_package = format!(
-                        "{} {}",
-                        format!("[{} {}]", unit.pkg.name(), unit.pkg.version()),
-                        warning
-                    );
+                    let warning_with_package =
+                        format!("{}@{} {}", unit.pkg.name(), unit.pkg.version(), warning);
```

Comment:
Could you put a `:` separator in there?  Seems like it'd help to distinguish this from the build scripts output.  I had it in my message but it can easily be lost in everything else I was saying :)

Reply:
I let this one slide, but now it's fixed :)